webrtc.html

```diff
@@ -1272,7 +1272,8 @@
                           </li>
                           <li>
                             <p><a>Create an RTCRtpTransceiver</a> with
-                            <var>sender</var> and <var>receiver</var>, and
+                            <var>sender</var>, <var>receiver</var> and
+                            "inactive" as direction, and
```

Comment:
Ah yes, I initialized to "inactive" to make the subsequent steps below work where it diffs against this direction to take action. But as you point out below, I should be using `currentDirection` below instead. Will fix.